source/content.js

```diff
@@ -31,8 +31,12 @@ function isPackageJson() {
 }
 
 function addHeaderLink(box, name, url) {
+  isGitLab() ?
   box.firstElementChild.prepend(doma(`
     <a class="btn btn-sm BtnGroup-item" href="${url}">${name}</a>
+  `)) :
+  box.querySelector('.Box-header').append(doma(`
+    <a class="btn btn-sm" href="${url}">${name}</a>
```

Comment:
`BtnGroup-item` is required or else the buttons appear as:

<img width="996" alt="split" src="https://user-images.githubusercontent.com/1402241/77782726-0619ab80-7058-11ea-939f-1af27427a931.png">

I'd avoid this ternary. `querySelector` can work in both cases, with the right selector